### PR TITLE
Ml4a.net

### PR DESCRIPTION
**Pull request by squinard**
I am not sure why the accuracy is so low, but I am assuming it is the dataset + the number of epochs + learning rate. It should be pretty easy to switch the dataset for any that you may like. While I tried gdown, there is definitely something rotten in their tar.gz. Anyway, anything at this point is a relatively simple fix. Enjoy!

### CODE REVIEW

**Comment by squinard** [comment]
This is the updated Reinforcement Learning section. I am not sure if I am PRing correctly though. Remember that we still haven't fixed the display, but the model should work.
All the best,
Sebastian

**Comment by genekogan** [comment]
Did you try any of the proposed solutions to rendering [here](https://stackoverflow.com/questions/52726475/display-openai-gym-in-jupyter-notebook-only)? Sounds like there's a few solutions, including what looks like just rendering it with straight html (which we could always wrap as a function in the ml4a backend, like [here for example](https://github.com/ml4a/ml4a/blob/0ed4162a51803aa185ec0fafab5cc3acdcb44e55/ml4a/image.py#L186-L195).

Another thing to consider is whether or not we should add `gym[atari]` as a dependency. The pro is that everything that you need installs on `pip install ml4a` (when it's on PyPi), the con is that the dependencies become really large and if any of them break on your own install, it screws up the whole install. A possible solution to this is to allow [multiple different installs of ml4a using the brackets notation](https://stackoverflow.com/questions/46775346/what-do-square-brackets-mean-in-pip-install).

**Comment by squinard** [comment]
I had not seen that solution, but it looks much better than the others I had found which would essentially involve a second script. It has been a while since I last looked at the notebook though. I will implement it the next time I am at my computer.
As to the second point, I think that having multiple installs definitely decreases user friendliness, but so does not being able to install the software lol. That is a tough decision.

**Comment by squinard** [comment]
Alright, I actually was able to implement the code in the question, rather than the answer, which was much more streamlined. Now, it plays the model on click! If you would like it to generate a loop, let me know, but this seems costly as it has to store all the pictures.
Best,
Sebastian

**Comment by genekogan** [comment]
This is great. I am going to merge.

Some non-urgent improvements that can be made in the future.
1) Getting it to play more accurately/reliably, perhaps show another runtime after a second, longer training period to improve.
2) Plot loss
3) More commentary/explaining for beginners, e.g. what is scene extraction, why do we use raw pixels instead of something else, etc.
